pyvrp/plotting/tests/test_plotting.py

```diff
@@ -52,9 +59,13 @@ def test_plot_result():
 
     data = read("data/RC208.txt", "solomon", round_func="trunc")
     bks = read_solution("data/RC208.sol")
+
     pm = PenaltyManager(data.vehicle_capacity)
     rng = XorShift128(seed=42)
-    pop = Population(data, pm, rng, broken_pairs_distance)
+    params = PopulationParams()
+    init = make_random_solutions(params.min_pop_size, pm, rng, data)
+
+    pop = Population(init, broken_pairs_distance, params=params)
     stats = Statistics()
```

Comment:
The spacing here doesn't make much sense to me. I'd expect `params`, `init`, and `pop` to all be together in the same paragraph (since they're all part of "population stuff").